docs/en_US/Compression/Tutorial.rst

```diff
@@ -23,24 +23,29 @@ There are different keys in a ``dict``. Some of them are common keys supported b
 
 Some other keys are often specific to a certain algorithm, users can refer to `pruning algorithms <./Pruner.rst>`__ and `quantization algorithms <./Quantizer.rst>`__ for the keys allowed by each algorithm.
 
-A simple example of configuration is shown below:
+To prune all ``Conv2d`` layers with the sparsity of 0.6, the configuration can be write as:
+.. code-block:: python
```

Comment:
Please double-check the rendering results.

Reply:
Thanks for point out that. Please review the latest version.